src/utility.jl

```diff
@@ -0,0 +1,27 @@
+function area(s::Triangle)
```

Comment:
I don't like having files called `utils` or `utility`, they quickly attract too much bad code. I'd suggest adding these methods to screen.jl (and maybe, later renaming _screen.jl_ to _geom.jl_ or some such)